code/game/machinery/door_control.dm

```diff
@@ -218,3 +217,17 @@
 	for (var/obj/machinery/door/window/plasma/secure/interogation_room/W in range(range))
 		if (W.smartwindow && src.id_tag == W.smartwindow.id_tag)
 			W.smartwindow.toggle_smart_transparency()
+
+/obj/machinery/door_control/mapped/box_armoury
+	name = "windoor control"
+	desc = "Open or close the windoors in the armoury."
+	id_tag = "IDTagBoxArmoury"
+	req_access = list(access_security)
```

Comment:
shouldn't it be access_armory?

Reply:
The windoors themselves are security access (an officer can open them).